### DISCUSSION TITLE
FigJam Link

### DISCUSSION THREAD

**Discussion by kagomen** [lgtm-factory/lgtm-factory#61] [Announcements]
## 概要

デザイン周りのイシューに参加したい人の、デザイン作業スペースです：
https://www.figma.com/board/bMwsq6U4qOEILPUk4E15NZ/LGTM-Factory?node-id=0-1&t=Dt8mxzkAf20upMmC-1

- #52 
上記に記載されている、サイトのデザインの工程です。

FigJamで、決まったことは、
Discord、issue、その他ドキュメントに記載して、メンバー全員に共有されます！

なので、FigJamを全て確認する必要はないです！
デザインの工程に、興味がある方は、参加してくださいー！🙌

**Comment by kazzyfrog** [comment]
ログインや、招待申請？みたいなの、かけといた方がいいかもです！

自分さっきログインせずに参加してしまったので！
ログインせず、色々できてしまう？と、色々ミスってしまいそう、かなと思いました！！🧐

**Comment by siso001** [comment]
## 「コンセプトキーワード」、「コンセプト」用語の認識の整理
「コンセプトキーワード」、「コンセプト」の認識が曖昧だったので、整理して確認しました！

「コンセプト」の認識
- サイトのイメージ・世界観を表す、**デザイン**のコンセプト
- サイト全体の方向性を示す、**要求**（サイト、サービス）のコンセプト
この二つがあります！色々考えているとごっちゃになるので注意が必要です

デザインを決めていく流れで決めるコンセプトキーワードは、デザインの雰囲気を一言で表すものなので、一つ目の認識です

明日以降の流れは、
- プロジェクト名を再考する
↓
- プロジェクト名と要求のコンセプトまとめから、デザインのコンセプトキーワードを決める
↓
- イメージソースを収集し、ムードボードを作成する #49 
...
という感じですかね？

**Reply by kagomen** [reply]
@siso001 

まとめありがとうございます！！🙏

> プロジェクト名を再考する
↓
プロジェクト名と要求のコンセプトまとめから、デザインのコンセプトキーワードを決める
↓
イメージソースを収集し、ムードボードを作成する

そんな感じだと思います！👍

加えて、要求のコンセプトがもう少し深掘りできそうな感じ（具体的には、チームやコミュニティ、コミュニケーションに"どう"フォーカスを当てるか？など）がありそう？なので、
プロジェクト名を再考したり、プロジェクト名を決める方法を探ったりしつつ、コンセプトもちょっと視点を変えて考えてみる、みたいな、ちょっと抽象的な感じがもう少し続きそうかな？と、思いました！

**Comment by siso001** [comment]
私的コンセプトキーワード案
- LGTeaM - Factory 👷
-

**Comment by kagomen** [comment]
![thumbs-1723634977549](https://github.com/user-attachments/assets/2267116a-83be-4c4b-bf57-7339bc561a6a)
![thumbs-1723635007280](https://github.com/user-attachments/assets/9927f26c-eb74-49d6-80ca-f273fa155a02)

**Reply by kagomen** [reply]
![image](https://github.com/user-attachments/assets/3a233478-603f-41a2-bf59-3d2675b5ae96)

**Reply by kagomen** [reply]
<img width="1149" alt="Group 2" src="https://github.com/user-attachments/assets/5215a226-3214-40c4-b3f2-fa30944892cc">
